src/table/table.component.ts

```diff
@@ -168,244 +168,249 @@ import { I18n } from "./../i18n/i18n.module";
-							[attr.aria-label]="(column.sorted && column.ascending ? sortDescendingLabel : sortAscendingLabel) | async"
-							aria-live="polite"
-							[ngClass]="{
-								'bx--table-sort-v2--active': column.sorted,
-								'bx--table-sort-v2--ascending': column.ascending
-							}"
-							(click)="sort.emit(i)">
-							<span
-								*ngIf="!column.template"
-								[title]="column.data"
-								tabindex="-1">
-								{{column.data}}
-							</span>
-							<ng-template
-								[ngTemplateOutlet]="column.template" [ngTemplateOutletContext]="{data: column.data}">
-							</ng-template>
-							<svg
-							class="bx--table-sort-v2__icon"
-							width="10" height="5" viewBox="0 0 10 5">
-								<path d="M0 0l5 4.998L10 0z" fill-rule="evenodd" />
-							</svg>
-						</button>
-						<span
-							class="bx--table-header-label"
-							*ngIf="!skeleton && this.sort.observers.length === 0 || (this.sort.observers.length > 0 && !column.sortable)">
-							<span *ngIf="!column.template" [title]="column.data">{{column.data}}</span>
-							<ng-template
-								[ngTemplateOutlet]="column.template" [ngTemplateOutletContext]="{data: column.data}">
-							</ng-template>
-						</span>
-						<button
-							[ngClass]="{'active': column.filterCount > 0}"
-							*ngIf="column.filterTemplate"
-							type="button"
-							aria-expanded="false"
-							aria-haspopup="true"
-							[ibmTooltip]="column.filterTemplate"
-							trigger="click"
-							[title]="filterTitle | async"
-							placement="bottom,top"
-							[data]="column.filterData">
-							<svg
-								xmlns="http://www.w3.org/2000/svg"
-								class="icon--sm"
-								width="16"
-								height="16"
-								viewBox="0 0 16 16">
-								<path d="M0 0v3l6 8v5h4v-5l6-8V0H0zm9 10.7V15H7v-4.3L1.3 3h13.5L9 10.7z"/>
-							</svg>
-							<span *ngIf="column.filterCount > 0">
-								{{column.filterCount}}
-							</span>
-						</button>
-						<div
-						*ngIf="columnsDraggable && isColumnDragging"
-						class="drop-area">
-							<div
-							*ngIf="columnDraggedHoverIndex == i && columnDraggedPosition == 'left'"
-							class="drop-indicator-left"></div>
-							<div
-							class="drop-area-left"
-							(dragenter)="columnDragEnter($event, 'left', i)"
-							(dragleave)="columnDragLeave($event, 'left', i)"
-							(dragover)="columnDragover($event, 'left', i)"
-							(drop)="columnDrop($event, 'left', i)">
-							</div>
-
-							<div
-							class="drop-area-right"
-							(dragenter)="columnDragEnter($event, 'right', i)"
-							(dragleave)="columnDragLeave($event, 'right', i)"
-							(dragover)="columnDragover($event, 'right', i)"
-							(drop)="columnDrop($event, 'right', i)">
-							</div>
-							<div
-							*ngIf="columnDraggedHoverIndex == i && columnDraggedPosition == 'right'"
-							class="drop-indicator-right"></div>
-						</div>
+	<div class="bx--data-table-v2-container">
+		<h4 class="bx--data-table-v2-header">{{title}}</h4>
```

Comment:
It's not clear to me that the title _needs_ to be baked into the table ... what does this get us?

Reply:
The whole table should be wrapped with the class `bx--data-table-v2-container` and since the title and toolbar are part of the table I thought it made sense to make it part of the component. Not sure if that's what we want to do though

Reply:
If none of the core table classes depend on being within `bx--data-table-v2-container` I think this would be better as another component ... that way consumers aren't locked into one way of using tables. We can even provide a `tableHeader` directive that applies the `bx--data-table-v2-header` class - that way consumers can totally customize their headers.